edk2toollib/utility_functions.py

```diff
@@ -103,9 +103,9 @@ def GetHostInfo():
     arch = None
     bit = None
 
-    if ("x86" in processor_info) or ("AMD" in processor_info) or ("Intel" in processor_info):
+    if ("x86" in processor_info.lower()) or ("AMD" in processor_info.upper()) or ("Intel" in processor_info):
```

Comment:
why is "Intel" not uppercase?

Reply:
should we also normalize 
host_info = platform.uname()
    os = host_info.system	    os = host_info.system
    processor_info = host_info.machine	    processor_info = host_info.machine

This might be a breaking change.  I know we have code that is doing os == "Linux"

Reply:
Are you saying we should normalize the OS?

Reply:
and host info?

Reply:
I think you could make the argument either way, since it's returning what Python provides and the docs have no firm guidance and kind of just say "here's what your system says it is."

We could take a firm stance and standardize to all lower case or all upper. I lean towards not modifying what the python system returns.